src/main/java/uk/gov/hmcts/sscs/service/SubmitAppealService.java

```diff
@@ -1,72 +1,102 @@
     private final EmailService emailService;
-    private SubmitYourAppealEmail submitYourAppealEmail;
+    private final SubmitYourAppealEmail submitYourAppealEmail;
 
     @Autowired
-    public SubmitAppealService(@Value("${appellant.appeal.html.template.path}") String appellantTemplatePath,
-                               PDFServiceClient pdfServiceClient,
-                               EmailService emailService,
-                               SubmitYourAppealEmail submitYourAppealEmail) {
+    SubmitAppealService(@Value("${appellant.appeal.html.template.path}") String appellantTemplatePath,
+                        AppealNumberGenerator appealNumberGenerator,
+                        SubmitYourAppealToCcdCaseDataDeserializer submitYourAppealToCcdCaseDataDeserializer,
+                        CcdService ccdService,
+                        PDFServiceClient pdfServiceClient,
+                        EmailService emailService,
+                        SubmitYourAppealEmail submitYourAppealEmail) {
 
         this.appellantTemplatePath = appellantTemplatePath;
+        this.appealNumberGenerator = appealNumberGenerator;
+        this.submitYourAppealToCcdCaseDataDeserializer = submitYourAppealToCcdCaseDataDeserializer;
+        this.ccdService = ccdService;
         this.pdfServiceClient = pdfServiceClient;
         this.emailService = emailService;
         this.submitYourAppealEmail = submitYourAppealEmail;
     }
 
     public void submitAppeal(SyaCaseWrapper appeal) {
+        CaseData caseData = transformAppealToCaseData(appeal);
+        CaseDetails caseDetails = createCaseInCcd(caseData);
+        byte[] pdf = generatePdf(appeal, caseDetails.getId());
+        sendPdfByEmail(appeal, pdf);
+    }
+
+    private CaseDetails createCaseInCcd(CaseData caseData) {
+        try {
+            return ccdService.createCase(caseData);
+        } catch (CcdException ccdEx) {
+            return CaseDetails.builder().build();
+        }
+    }
+
+    private CaseData transformAppealToCaseData(SyaCaseWrapper appeal) {
+        CaseData caseData = submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal);
+        Subscription subscription = caseData.getSubscriptions().getAppellantSubscription().toBuilder()
+                .tya(appealNumberGenerator.generate())
+                .build();
+        caseData.getSubscriptions().setAppellantSubscription(subscription);
+        return caseData;
+    }
+
+    private void sendPdfByEmail(SyaCaseWrapper appeal, byte[] pdf) {
         String appellantLastName = appeal.getAppellant().getLastName();
         String nino = appeal.getAppellant().getNino();
-
         String appellantUniqueId = String.format(ID_FORMAT, appellantLastName, nino.substring(nino.length() - 3));
+        submitYourAppealEmail.setSubject(appellantUniqueId);
+        submitYourAppealEmail.setAttachments(newArrayList(pdf(pdf, appellantUniqueId + ".pdf")));
+        emailService.sendEmail(submitYourAppealEmail);
```

Comment:
AppInsight alert missing when emailService is not available.

Reply:
It's not missing. There is an uchecked exception that will be triggered if there is any issue when sending an email. That will generate an AppInsight alert. Don't see any issue here. Same as above comment, really.

Reply:
If we receive alert then thats fine.